README.md

```diff
@@ -1,503 +1,62 @@
-
-<style>
-  body #__nuxt, #__layout,#defaultLayout {
-    display: flex;
-    min-height: 98vh;
-    flex-direction: column;
-  }
-  .content {
-    flex: 1;
-  }
-</style>
 ```
-
-Now you have common elements which can be displayed around your pages. You can use different layouts to handle a change in presentation between different parts of your site from marketing to documentation to tutorials.
-
-Currently, your application is looking pretty bland so let's add some CSS. Inside the assets directory create a new folder called `css` and add a `main.css` file with the following content:
-
+npm install
+npm run dev
 ```
-html{
-    background: #20313b;
-}
-
-body{
-    font-family: 'Roboto', sans-serif;
-    color: hsl(240, 5%, 83%);
-}
 
-.content{
-    margin-top: 50px;
-    display: flex;
-    flex-direction: column;
-    flex-wrap: wrap;
-    justify-content: center;
-    align-items: center;
-    font-weight: lighter;
-}
+This will start the application in development mode so any of your changes trigger a live reload.
 
-footer{
-    font-size: smaller;
-    display: flex;
-    flex-direction: column;
-    flex-wrap: wrap;
-    justify-content: center;
-    align-items: center;
-    font-weight: lighter;
-}
+## Connect Okta
 
-header{
-    display: flex;
-    flex-direction: column;
-    flex-wrap: wrap;
-    justify-content: center;
-    align-items: center;
-    font-weight: lighter;
-}
+Okta has Authentication and User Management APIs that reduce development time with instant-on, scalable user infrastructure. Okta's intuitive API and expert support make it easy for developers to authenticate, manage and secure users and roles in any application.
 
-#branding{
-    font-family: 'Berkshire Swash', cursive;
-}
+This sample uses Okta to manage users and authenticate them to protected routes on the application. Before you begin, you'll need a free Okta developer account. Install the [Okta CLI](https://cli.okta.com/) and run ```okta register``` to sign up for a new account. If you already have an account, run ```okta login```.
 
-.statement{
-    color: white;
-    font-size: 350%
-}
+Then, run ```okta apps create```. Select the default app name, or change it as you see fit. Choose Single-Page App and press Enter. 
 
-.callToAction {
-    background-color: #e22866;
-    color: white;
-    border-radius: 4px;
-    font-size: 14px;
-    font-weight: 600;
-    letter-spacing: 2px;
-    line-height: 45px;
-    height: 45px;
-    border: 6px solid transparent;
-    cursor: pointer;
-    text-decoration: none;
-}
+Change the Redirect URI to http://localhost:3000/login and accept the default Logout Redirect URI of http://localhost:3000.
 
-.userAgent{
-    color: #e22866
-}
+The Okta CLI will create an OIDC Single-Page App in your Okta Org. It will add the redirect URIs you specified and grant access to the Everyone group. It will also add a trusted origin for http://localhost:3000. You will see output like the following when it's finished:
```

Comment:
```suggestion
The Okta CLI will create an OIDC Single-Page App in your Okta Org. It will add the redirect URIs you specified and grant access to the Everyone group. It will also add a trusted origin for `http://localhost:3000`. You will see output like the following when it's finished:
```